app/configurations/config.hb.js

```diff
@@ -8,6 +8,7 @@ const API_URL = process.env.API_URL || 'https://api.mobil-in-herrenberg.de';
 const MAP_URL = process.env.MAP_URL || 'https://maps.wikimedia.org/osm-intl/';
 const GEOCODING_BASE_URL = process.env.GEOCODING_BASE_URL || `https://pelias.locationiq.org/v1`;
 const LOCATIONIQ_API_KEY = process.env.LOCATIONIQ_API_KEY;
+const GEOJSON_LAYERS_BASE_URL = process.env.GEOJSON_LAYERS_BASE_URL || 'http://opentripplanner-data-con-hb:8080/routing-data/v2/hb/';
```

Comment:
This won't work, as it is a URL delivered to the client, and not interpreted on digitransit server host.

Reply:
I changed this to the local path and kept the variable too, is it ok?